app/models/spree/retail/shopify/generate_refund_order.rb

```diff
@@ -8,9 +8,20 @@ def initialize(shopify_refund)
         end
 
         def process
+          if order.returned?
+            logger.info("SKIP - [Spree: #{@order.number} / Shopify: #{@order.pos_order_number}] - refund already imported")
+            return true
+          end
+
           create_return_authorization(order, return_items)
           customer_return = create_customer_return(return_items)
           create_reimbursement(customer_return)
+
+          return true
+
+        rescue => e
```

Comment:
Any concern about indiscriminately rescuing all exceptions, vs. selectively rescuing exceptions you anticipate?